client-runtime/src/main/java/com/microsoft/rest/v2/http/SharedChannelPool.java

```diff
@@ -42,9 +45,11 @@
  */
 class SharedChannelPool implements ChannelPool {
     private static final AttributeKey<URI> CHANNEL_URI = AttributeKey.newInstance("channel-uri");
+    private static final AttributeKey<ZonedDateTime> CHANNEL_AVAILABLE_SINCE = AttributeKey.newInstance("v2runtime_reserved_channel-available-since");
```

Comment:
a shorter name and use '-' instead of '_'?

Reply:
done